### DISCUSSION TITLE
What is Wails path on mac

### DISCUSSION THREAD

**Discussion by nabeelaejaz** [wailsapp/wails#1478] [Q&A] [answered]
Hello, I am just starting out here. I installed wails successfully on windows. But on Mac, my terminal is unable to locate Wails. Can you please tell me what is the path to wails on mac, so that I may include it in the profile. Thanks.

**Comment by leaanthony** [comment, marked as answer]
Same as all go apps: `~/go/bin`

**Comment by nabeelaejaz** [comment]
wails doctor .. now gives me this error. Kindly help:

ERROR: exec: "sysctl": executable file not found in $PATH

**Reply by leaanthony** [reply]
Your paths are broken. Reinstall MacOS.
